Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/TapToPlace/TapToPlace.cs

```diff
@@ -0,0 +1,467 @@
+
+        /// <summary>
+        /// The max distance to place an object if there is a raycast hit on a surface
+        /// </summary>
+        public float MaxRaycastDistance
+        {
+            get => maxRaycastDistance;
+            set => maxRaycastDistance = value;
+        }
+
+        [SerializeField]
+        [Tooltip("Array of LayerMask to execute from highest to lowest priority. First layermask to provide a raycast hit will be used by component")]
+        private LayerMask[] magneticSurfaces = { UnityEngine.Physics.DefaultRaycastLayers };
+
+        /// <summary>
+        /// Array of LayerMask to execute from highest to lowest priority. First layermask to provide a raycast hit will be used by component.
+        /// </summary>
+        public LayerMask[] MagneticSurfaces
+        {
+            get => magneticSurfaces;
+            set => magneticSurfaces = value;
+        }
+
+        //[SerializeField]
+        //[Tooltip("Is the gameObjectToPlace currently in a state where it is being placed? This state is activated when you selecthe GameObjectToPlace.")]
+        //private bool isBeingPlaced = false;
+
+        /// <summary>
+        /// Is the gameObjectToPlace currently in a state where it is being placed? This state is activated when you select
+        /// the GameObjectToPlace.
+        /// </summary>
+        public bool IsBeingPlaced { get; protected set; }
+
+        [SerializeField]
+        [Tooltip("If true, the gameObjectToPlace will start in the placing state.  Meaning that the object will immediately start" +
+        "by following the TrackedTargetType (Head or Controller Ray) and then a tap is required to place the object.  AutoStart" +
+         "is only called once because it is in Start();")]
+        private bool autoStart = false;
+
+        /// <summary>
+        /// If true, the gameObjectToPlace will start in the placing state.  Meaning that the object will immediately start
+        /// by following the TrackedTargetType (Head or Controller Ray) and then a tap is required to place the object.  AutoStart
+        /// is only called once because it is in Start();
+        /// </summary>
+        public bool AutoStart
+        {
+            get => autoStart;
+            set
+            {
+                if (autoStart != value)
+                {
+                    autoStart = value;
+                }
+            }
+        }
+
+        /// <summary>
+        /// If true, the raycast did hit a surface
+        /// </summary>
+        public bool DidHit { get; protected set; }
+
+        [SerializeField]
+        [Tooltip("The distance between the center of the gameobject to place and a surface along the surface normal, if the raycast hits a surface")]
+        private float surfaceNormalOffset = 0.0f;
+
+        /// <summary>
+        /// SurfaceNormalOffset is the offset between the game object's center and The default value for SurfaceNormalOffset is half the depth of a game object. 
```

Comment:
```suggestion
        /// The offset between the game object's center and The default value for SurfaceNormalOffset is half the depth of a game object. 
```

Reply:
Or just update to be consistent with the comment above.

Reply:
Updated comment to be consistent with tooltip above